libs/testnorge-domain/src/main/java/no/nav/registre/testnorge/domain/dto/arena/testnorge/brukere/Arbeidsoeker.java

```diff
@@ -23,4 +23,5 @@
     private Boolean automatiskInnsendingAvMeldekort;
     private Boolean aap115;
     private Boolean aap;
+    private String oppfolging;
```

Comment:
Veeldig liten greie, men ser det brukes oppfoelging ellers i koden 🤓

Reply:
Ja, det heter oppfolging i arena så det må stå det her. Men siden vi generelt bruker oe for ø i resten av koden brukte jeg oppfoelging også.